drivers/starburst/resources/metabase-plugin.yaml

```diff
@@ -37,6 +37,11 @@ driver:
       display-name: Role (optional)
       required: false
       helper-text: Specify a role to override the database user’s default role.
+    - name: impersonation
+      display-name: Use the Metabase user as Starburst user
```

Comment:
nit: Execute queries using the Metabase user

Reply:
I asked Piotr what the text should actually be